### DISCUSSION TITLE
新玩具-极空间Z4pro | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#146] [Announcements]
https://blog.goalonez.site/blog/%E6%96%B0%E7%8E%A9%E5%85%B7-%E6%9E%81%E7%A9%BA%E9%97%B4Z4pro.html 

 Goalonez Blog

**Comment by Yinsuxiazhe** [comment]
大佬好，能不能求一个能导入镜像的包（tailscale），我也在组局域网，我的邮箱meetfcy@126.com，谢谢您！

**Comment by Goalonez** [comment]
> @Yinsuxiazhe
> 大佬好，能不能求一个能导入镜像的包（tailscale），我也在组局域网，我的邮箱meetfcy@126.com，谢谢您！

可以去找一些还能用的镜像站地址，比如dockerhub.icu之类的。或者自己本地科学拉到镜像save下来导进去。

**Comment by Yinsuxiazhe** [comment]
好的，谢谢大佬！



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年07月05日 12:12 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [Goalonez/goalonez.github.io] 新玩具-极空间Z4pro | Goalonez (Issue #84) |

@Yinsuxiazhe
***@***.***，谢谢您！

可以去找一些还能用的镜像站地址，比如dockerhub.icu之类的。或者自己本地科学拉到镜像save下来导进去。

**Comment by luxuxl** [comment]
博主, 你好请问手机相册的同步可以做到双向同步吗, 我目前体验的都是只会自动上传, 但如果相册删除了某个照片, 云空间是不会删除的

**Comment by Goalonez** [comment]
> @luxuxl
> 博主, 你好请问手机相册的同步可以做到双向同步吗, 我目前体验的都是只会自动上传, 但如果相册删除了某个照片, 云空间是不会删除的

不会删除，单向的。目前我也没用这个功能了。
